src/views/Overview/Overview.vue

```diff
@@ -101,47 +100,63 @@ export default {
     PageTitle,
     PageSection
   },
-  data() {
-    return {
-      logging: {
-        entry: {
-          Description:
-            'An internal failure has occurred while performing an operation.',
-          EventID: 'FQPSPCR0021F',
-          Id: 1,
-          Resolved: false,
-          Severity: 'xyz.openbmc_project.Logging.Entry.Level.Error',
-          Timestamp: 1574782085071,
-          Version: 'ibm-v2.7.0-rc1-5-gfd9b55f-r19-1-g8c075d3'
-        }
-      },
-      network: {
-        config: {
-          HostName: 'witherspoon'
-        },
-        eth0: {
-          MACAddress: '00:00:00:00:00:00'
-        },
-        ipv4: {
-          Address: '00.00.00.00'
-        }
-      },
-      power_cap: {
-        PowerCap: 0,
-        PowerCapEnable: false
-      },
-      software: {
-        Version: 'IBM-witherspoon-OP9-v2.4-4.22'
-      },
-      system: {
-        Manufacturer: '',
-        Model: '0000000000000000',
-        SerialNumber: '0000000000000000'
-      },
-      total_power: {
-        description: '0'
-      }
-    };
+  created() {
+    this.getServerInfo();
+    this.getHostInfo();
+    this.getFirmwareInfo();
+    this.getPowerData();
+    this.getPowerCapData();
+    this.getNetworkData();
+    this.getEventLogData();
+  },
+  computed: {
```

Comment:
can we use mapState here? https://vuex.vuejs.org/guide/state.html#the-mapstate-helper